product/versioning.md

```diff
@@ -0,0 +1,24 @@
+# Versioning
+
+## 1.0
+
+The 1.0 Release of FlowForge represents a significant milestone, we are making a statement to our customers about the product.
+
+Therefore the following are considerations & requirements in order to be able to stand behind that statement.
+
+- We have the admin features required to manage the platform at an appropriate scale.
+- We have the monitoring and alerting features to run a commercial SaaS platform.
+- We have a suitable licensing model in place
```

Comment:
Minor note: for me just project pricing might even suffice for 1.0. Devices would be great, teams IMO is a stretch goal.

Reason being that building a feature gap between tiers takes time. Building a gap on 3 axis is taking exponentially more time is my gut feeling.